PSReadLine/History.cs

```diff
@@ -677,6 +679,21 @@ private void HistorySearch(int direction)
             }
         }
 
+        private string GetHistorySuggestion(string text)
+        {
```

Comment:
I'd like to see comments here for possible improvements, e.g. not using StartsWith and instead extracting a partial pipeline/statement from a longer pipeline or multiline command.

Reply:
Will add the comment.